miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultRunService.java

```diff
@@ -554,7 +554,12 @@ public boolean processNotification(Run source, int laneCount, String containerMo
 
     SequencingContainerModel model = containerService.findModel(sequencer.getPlatform(), containerModel, laneCount);
     if (model == null) {
-      throw new IllegalArgumentException("Invalid container parameters: model=" + containerModel + ", lanes=" + laneCount);
+      throw new IllegalArgumentException(
+          "Could not find container or fallback for parameters: model=" + containerModel + ", lanes=" + laneCount);
+    }
+    if (model.isFallback() && isNew) {
+      log.info("Could not find container model with model=" + containerModel + " and lanes=" + laneCount
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)